doc/USB.md

```diff
@@ -27,3 +27,4 @@
 | Mustang LT 40S | 1ed8 | 0046 |      |
 | Rumble LT 25   | 1ed8 | 0038 |      |
 | Mustang GT 40  | 1ed8 | 0032 |      |
+| Mustang Micro  | 1ed8 | 0043 |      |
```

Comment:
Nice, could you submit this in a separate PR?

Reply:
Will do shortly.   Unfortunately, experimentation to date shows that the Mustang Micro is  completely unresponsive to the commands which are getting responses on the LT40S, so this may only be useful to put a disclaimer to say this device is not supportable.  It is not surprising, as the 'Micro is not interoperable with the Desktop Fender Tone app either.

Reply:
PR for this will come in in a couple of minutes.